Specs/DataSources/DataSourceCollectionSpec.js

```diff
@@ -53,6 +53,37 @@ defineSuite([
         expect(removeSpy).toHaveBeenCalledWith(collection, source);
     });
 
+    it('move event works', function() {
+        var source = new MockDataSource();
+        var collection = new DataSourceCollection();
+        collection.add(source);
+
+        var moveSpy = jasmine.createSpy('dataSourceMoved');
+        collection.dataSourceMoved.addEventListener(moveSpy);
+
+        collection.raise(source);
+        collection.lower(source);
+        collection.raiseToTop(source);
+        collection.lowerToBottom(source);
```

Comment:
We try to avoid `uber tests` like this that call a whole bunch of functions at once because they are hard to debug when something brakes.  If you think this is the best way to test the ordering functionality, we can keep this.  But please add comments to each function call indicating what exactly is being tested.